tweets/security-releases-2021-03-31.tweet

```diff
@@ -0,0 +1,8 @@
+The Node.js project will release new versions of all supported release lines on or shortly after Tuesday, April 6th, 2021.
```

Comment:
```suggestion
We will be releasing new versions of all supported release lines on or shortly after Tuesday, April 6th, 2021.
```

Reply:
I wouldn't say "Tierney is publishing a new version of temporalts tomorrow". Want to make sure we're talking in the voice of Node.js.